src/Mautic/Form.php

```diff
@@ -127,18 +130,18 @@ public function prepareRequest(array $data)
         $request['url'] = $this->getUrl();
         $request['data'] = ['mauticform' => $data];
 
-        if ($contactId = $contact->getId()) {
-            $request['data']['mtc_id'] = $contactId;
-        }
+        //if ($contactId = $contact->getId()) {
+            $request['data']['mtc_id'] =  $data['mtc_id'];  //$contactId;
+        //}
 
         if ($contactIp = $contact->getIp()) {
             $request['header'][] = "X-Forwarded-For: $contactIp";
             $request['header'][] = "Client-Ip: $contactIp";
         }
 
         if ($sessionId = $contact->getSessionId()) {
-            $request['header'][] = "Cookie: mautic_session_id=$sessionId";
-            $request['header'][] = "Cookie: mautic_device_id=$sessionId";
+//            $request['header'][] = "Cookie: mautic_session_id=$sessionId";
+//            $request['header'][] = "Cookie: mautic_device_id=$sessionId";
```

Comment:
What's wrong with this library to set the cookies for you? I think I'll need more context. And the changes need a bit of cleanup. Leaving commented out code in the production code isn't very professional.

Reply:
Hello @escopecz, thanks for the library, it helps us a lot. Hello @zaharovrd, we are facing the same problem you described. Submitting the form creates a new contact. Can you please share your solution to solve this problem?
![grafik](https://github.com/escopecz/mautic-form-submit/assets/40004786/71e2fa5b-370b-4a88-80d6-e14c592dafcb)

Reply:
This is correct repo:
https://github.com/zaharovrd/mautic-form-submit